connect/api/src/main/java/org/apache/kafka/connect/connector/policy/ConnectorClientConfigOverridePolicy.java

```diff
@@ -23,25 +23,25 @@
 import java.util.List;
 
 /**
- * <p>An interface for enforcing a policy on overriding of client configs via the connector configs.
- *
- * <p>Common use cases are ability to provide principal per connector, <code>sasl.jaas.config</code>
+ * An interface for enforcing a policy on overriding of Kafka client configs via the connector configs.
+ * <p>
+ * Common use cases are ability to provide principal per connector, <code>sasl.jaas.config</code>
  * and/or enforcing that the producer/consumer configurations for optimizations are within acceptable ranges.
  */
 public interface ConnectorClientConfigOverridePolicy extends Configurable, AutoCloseable {
 
 
     /**
-     * Worker will invoke this while constructing the producer for the SourceConnectors,  DLQ for SinkConnectors and the consumer for the
-     * SinkConnectors to validate if all of the overridden client configurations are allowed per the
-     * policy implementation. This would also be invoked during the validation of connector configs via the Rest API.
-     *
+     * Workers will invoke this while constructing producer for SourceConnectors, DLQs for SinkConnectors and
+     * consumers for SinkConnectors to validate if all of the overridden client configurations are allowed per the
```

Comment:
We probably don't need to enumerate all of the use-cases for kafka clients here, and can keep that scoped to the ConnectorClientConfigRequest javadoc.
```suggestion
     * Workers will invoke this before configuring per-connector Kafka admin, producer, and consumer
     * client instances to validate if all of the overridden client configurations are allowed per the
```

Reply:
Makes sense